cmd/state-offline-installer/runInstall.go

```diff
@@ -77,41 +78,55 @@ func runInstall(out output.Outputer, params *Params) error {
 		return errs.Wrap(err, "Unable to create artifactsPath directory")
 	}
 
-	err = unzip(params.backpackZipFile, assetsPath)
+	ua := unarchiver.NewZip()
+	out.Print(fmt.Sprintf("Stage 1 of 3 Start: Decompressing assets into: %s", assetsPath))
+	f, siz, err := ua.PrepareUnpacking(params.backpackZipFile, assetsPath)
 	if err != nil {
-		return errs.Wrap(err, "Unable to unzip to assetsPath")
+		return errs.Wrap(err, "Unable to prepare unpacking of backpack")
 	}
 
-	eulaAccepted, err := showEULAAndGetAcceptance(&prompter, licenseFilePath)
+	err = ua.Unarchive(f, siz, assetsPath)
 	if err != nil {
-		return errs.Wrap(err, "Error with EULA acceptance")
+		return errs.Wrap(err, "Unable to unarchive Assets to assetsPath")
 	}
 
-	if !eulaAccepted {
-		return locale.NewInputError("err_eula_not_accepted", "End user didn't accept the EULA")
-	}
+	out.Print(fmt.Sprintf("Stage 1 of 3 Finished: Decompressing assets into: %s", assetsPath))
 
-	out.Print(fmt.Sprintf("Stage 1 of 2 Start: Decompressing assets into: %s", assetsPath))
-	err = unzip(params.backpackZipFile, assetsPath)
+	tos := prompts.NewOfflineFileTOS(licenseFilePath)
+	accepted, err := prompts.PromptTOS(tos, out, prompt)
```

Comment:
Nit: I suggest simplifying this to something like:

```
accepted, err := prompts.PromptOfflineFileTOS(out, prompt)
```

Which can under the hood do what you have here.